fili-core/src/main/java/com/yahoo/bard/webservice/table/PermissiveConcretePhysicalTable.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2017 Yahoo Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.table;
+
+import com.yahoo.bard.webservice.data.config.names.TableName;
+import com.yahoo.bard.webservice.data.time.ZonedTimeGrain;
+import com.yahoo.bard.webservice.table.availability.Availability;
+
+import javax.validation.constraints.NotNull;
+import java.util.Map;
+import java.util.Set;
+
+/**
+ * An instance of permissive Physical table that is backed by a single fact table.
+ */
+public class PermissiveConcretePhysicalTable extends ConcretePhysicalTable {
+    /**
+     * Create a permissive concrete physical table.
+     *
+     * @param name  Name of the physical table as TableName
+     * @param timeGrain  time grain of the table
+     * @param columns  The columns for this table
+     * @param logicalToPhysicalColumnNames  Mappings from logical to physical names
+     * @param availability the instance of Availability specifying the availabilities of table columns
+     */
+    protected PermissiveConcretePhysicalTable(
+            @NotNull TableName name,
+            @NotNull ZonedTimeGrain timeGrain,
+            @NotNull Set<Column> columns,
+            @NotNull Map<String, String> logicalToPhysicalColumnNames,
+            @NotNull Availability availability
```

Comment:
This is the exact same constructor as exists in `ConcretePhysicalTable`, and there are otherwise no methods in this class that are not directly from the parent. What added value is this class giving us?

As a minimum, I would expect that the constructor here would require a `PermissiveAvailability`, rather than _any_ `Availability`, otherwise there's no assertion that this `PhysicalTable` is going to do anything "permissive".